src/EnergyPlus/InternalHeatGains.cc

```diff
@@ -6309,10 +6322,14 @@ namespace InternalHeatGains {
                     TAirIn = TRecirc * RecircFrac + TSupply * (1.0 - RecircFrac);
                     WAirIn = WRecirc * RecircFrac + WSupply * (1.0 - RecircFrac);
                 } else if (AirConnection == ITEInletRoomAirModel) {
-                    // Room air model option not implemented yet
+                    // Room air model option: TAirIn=TAirZone, according to EngineeringRef 17.1.4
                     TAirIn = MAT(NZ);
+                    TSupply = TAirIn;
                     WAirIn = ZoneAirHumRat(NZ);
-                } else { // Default to ITEInletZoneAirNode
+                } else {
+                    // TAirIn = TRoomAirNodeIn, according to EngineeringRef 17.1.4
+                    int ZoneAirInletNode = DataZoneEquipment::ZoneEquipConfig(NZ).InletNode(1);
+                    TSupply = Node(ZoneAirInletNode).Temp;
```

Comment:
Applying clang format resulted in quite a lot of changes, but this section is one of the functional changes.